### PR TITLE
[14.0][ADD] partner_identification_sale.

### PR DESCRIPTION
**Pull request by Chanakya-OSI**

### CODE REVIEW

File: partner_identification_sale/README.rst
Comment (rousseldenis) [inline partner_identification_sale/README.rst:28]: @Chanakya-OSI From your description, I think pulling the stock module is a little bit too much no  ?

Comment (patrickrwilson) [reply partner_identification_sale/README.rst:28]: @rousseldenis the stock module adds the page/sections to the product so this module would add to that which is why it's in the depends. Future modules for manufacturing and such would do the same.

Comment (rousseldenis) [reply partner_identification_sale/README.rst:28]: I understand, what I wanted to says, is this module that is named ..._sale should depend on stock ?

As Odoo did a sale_stock glue module, shouldn't we do the same and have a partner_identification_sale_stock?

We can imagine someone that has only sale installed and want to restrict sale confirmation with Ids (and does not want to install stock as it requires new fees for instance).

Comment (patrickrwilson) [reply partner_identification_sale/README.rst:28]: @rousseldenis I understand what you are saying now and agree this is a better approach. @Chanakya-OSI lets move forward with this suggestion. I think it would just be a matter of removing the stock dependency and changing the product views to be similar to the stock module where if you install this alone then the options would be on the products. Then create a new module called product.template.attribute.value changing the view so they coincide together. Please reach out to me if you have questions.

**Comment by patrickrwilson** [comment]
Depends on #1127

**Comment by patrickrwilson** [comment]
@Chanakya-OSI works better with last commit. Just need to fix the test as it looks like it's still referring to a stock field and we should be good to go.

**Review by patrickrwilson — APPROVED** [review]
This works well, thank you. LGTM once #1127 merges.

**Comment by patrickrwilson** [comment]
@rousseldenis do you mind reviewing?

**Comment by rousseldenis** [comment]
Same remarks as for https://github.com/OCA/partner-contact/pull/1122

**Pull request closed without merging**
